src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -78,23 +79,40 @@ func MaxQemuVCPUs() uint32 {
 	return uint32(240)
 }
 
-func newQemuArch(config HypervisorConfig) qemuArch {
+func newQemuArch(config HypervisorConfig) (qemuArch, error) {
 	machineType := config.HypervisorMachineType
 	if machineType == "" {
 		machineType = defaultQemuMachineType
 	}
 
+	var mp *govmmQemu.Machine
+	for _, m := range supportedQemuMachines {
```

Comment:
duplicated code, please use the `for` of the line 103

Reply:
Good point.  Though since this is added in a later patch than this one, it's more the other way around.  Adjusted in next spin.